### PR TITLE
bugfixes

### PR DESCRIPTION
**Pull request by aledista**

### CODE REVIEW

File: malware/get_samples.py [hunk 1]
```diff
@@ -66,7 +66,7 @@ def run_query(url):
         # the response is a 'data' object, which is a list of objects
         resp = json.loads(urlopen(url).read())['data'][0]
     except Exception as e:
-        print('Failed to load "%s": %s' % (md5, str(e)))
+        print('Failed to load "%s": %s' % ('md5', str(e)))
```
Comment (mgoffin) [inline malware/get_samples.py]: I guess we could just include "md5" as part of the string instead of adding it in the tuple? Nice catch!

**Comment by mgoffin** [comment]
Thanks for cleaning all of this up! Other than the line note, everything else looks great!

File: malware/get_samples.py (hunk 1 above)
Comment (jessek) [inline malware/get_samples.py]: It's a nit, but it's a good nit. Let's move md5 into the string and I'll merge.

**Comment by jessek** [comment]
Nice catches. Did you find these because you triggered the exceptions?

Thank you for contributing to ThreatExchange!

**Comment by aledista** [comment]
Nope, just looking at the code :)

**Pull request merged**